modules/softwarelandscape/exercise3.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Role playing
+type: exercise
+order: 4
+---
+
+## Exercise: Role playing exercise (15 minutes)
```

Comment:
Nice, I think these role playing exercises are the right way to go for training support staff